ui/src/templates/playlist-timeline-header-info.hbs

```diff
@@ -0,0 +1,13 @@
+<h4 class="modal-title editor-modal-title">
+    Edit Playlist - {{ playlist.name }} -
```

Comment:
We need to keep the first part of this info to be translatable. You can create a new variable in `common.twig` for the playlist editor. Maybe `playlistEditorTrans`, and then pass that variable to `timelineHeaderInfoTemplate`